src/pages/algorithms/index.md

```diff
@@ -9,4 +9,4 @@ Some common types of algorithms include sorting algorithms, search algorithms, a
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/kPRA0W1kECg" frameborder="0" allowfullscreen></iframe>
 
-freeCodeCamp's curriculum heavily emphasizes creating algorithms. This is because learning algorithms is a good way to practice programming skills. Interviewers most commonly test candidates on algorithms during developer job interviews.
+freeCodeCamp's curriculum heavily emphasizes creating algorithms. This is because learning algorithms is a good way to practice programming skills. Interviewers most commonly test candidates on algorithms during developer job interviews due to the facts that a good program cannot be written without a good Algorithm. Algorithm is important to a programmer the way a house-plan is important to a bricklayer.
```

Comment:
@ibnahmadbello Can you correct facts to fact and lowercase the A for algorithm in the same sentence.